dev-lang/zig/zig-0.13.0-r1.ebuild

```
@@ -0,0 +1,242 @@
+# Copyright 2019-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LLVM_COMPAT=(18)
```

The style in Gentoo is to usually have whitespace around it:
```suggestion
LLVM_COMPAT=( 18 )
```

---

0fa14e036947d60c2afe4cac3c1c50669bb3722b